Modules/_testlimitedcapi/import.c

```diff
@@ -0,0 +1,285 @@
+// Need limited C API version 3.13 for PyImport_AddModuleRef()
+#include "pyconfig.h"   // Py_GIL_DISABLED
+#if !defined(Py_GIL_DISABLED) && !defined(Py_LIMITED_API)
+#  define Py_LIMITED_API 0x030d0000
+#endif
+
+#include "parts.h"
+#include "util.h"
+
+
+/* Test PyImport_GetMagicNumber() */
+static PyObject *
+pyimport_getmagicnumber(PyObject *Py_UNUSED(module), PyObject *Py_UNUSED(args))
+{
+    long magic = PyImport_GetMagicNumber();
+    return PyLong_FromLong(magic);
+}
+
+
+/* Test PyImport_GetMagicTag() */
+static PyObject *
+pyimport_getmagictag(PyObject *Py_UNUSED(module), PyObject *Py_UNUSED(args))
+{
+    const char *tag = PyImport_GetMagicTag();
+    return PyUnicode_FromString(tag);
+}
+
+
+/* Test PyImport_GetModuleDict() */
+static PyObject *
+pyimport_getmoduledict(PyObject *Py_UNUSED(module), PyObject *Py_UNUSED(args))
+{
+    return Py_XNewRef(PyImport_GetModuleDict());
+}
+
+
+/* Test PyImport_GetModule() */
+static PyObject *
+pyimport_getmodule(PyObject *Py_UNUSED(module), PyObject *name)
+{
+    assert(!PyErr_Occurred());
```

Comment:
Add `NULLABLE(name)` to be able to test with NULL.

Reply:
It does crash.

Reply:
This is great too. We can check (manually) that it crashes.

But some functions can raise SystemError for NULL, and _this_ should have unit tests.

Reply:
`PyImport_Import(NULL)` does raise SystemError, I added a test for this case.